docs/governance/community-guidelines/releases/versioning.md

```diff
@@ -25,7 +25,7 @@ Versioning will be scoped to each artifact delivered by the working groups. This
 
 ## Release Frequency
 
-Releases should happen, at most, one time per month. This schedule ensures a manageable release cadence and maintains the stability of our artifacts. For more information about the releases, please refer to this document: [Releases](./releases.md)
+Releases should happen, at most, one time per month. This schedule ensures a manageable release cadence and maintains the stability of our artifacts. For more information about the releases, please refer to this document: [Releases](release-standard/releases.md)
```

Comment:
```suggestion
Releases should happen, at most, one time per month. This schedule ensures a manageable release cadence and maintains the stability of our artifacts. For more information about the releases, please refer to this document: [Releases](/docs/governance/community-guidelines/releases.md)
```

Releases should happen, at most, one time per month. This schedule ensures a manageable release cadence and maintains the stability of our artifacts. For more information about the releases, please refer to this document: [Releases](/docs/governance/community-guidelines/releases.md)